lib/jormungandr/src/Cardano/Wallet/Network/Jormungandr.hs

```diff
@@ -0,0 +1,161 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC  -fno-warn-redundant-constraints #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+--
+-- A @NetworkLayer@ supporting jormungandr as a chain producer.
+
+module Cardano.Wallet.Network.Jormungandr
+    ( Jormungandr (..)
+    , mkNetworkLayer
+    , newNetworkLayer
+    , mkJormungandr
+
+    -- * Re-export
+    , BaseUrl (..)
```

Comment:
Some gratuitous re-exports here to avoid the dependencies in `.cabal` and `import`-statements for these tiny things.